javascript/src/main/pages/Home/Home.js

```diff
@@ -6,6 +6,7 @@ import { Redirect } from "react-router-dom";
 import BasicCourseSearchForm from "main/components/BasicCourseSearch/BasicCourseSearchForm";
 import JSONPrettyCard from "main/components/Utilities/JSONPrettyCard";
 import { fetchBasicCourseJSON } from "main/services/courseSearches";
+//import {fetchInstructorHistoryNameQtrJSON} from "main/services/courseSearches";
```

Comment:
Instead of a comment, it's probably cleaner to just delete it

Reply:
Yes, in this case, definitely